mpact_api/mpact/constants.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+class Constants:
+    ID = "id"
+    MESSAGE = "message"
+    CHAT = "chat"
+    FROM = "from"
+    TEXT = "text"
+    OK = "ok"
+    NEW_CHAT_PARTICIPANT = "new_chat_participant"
+    USERNAME = "username"
+    BOT_USERNAME = os.environ.get("BOT_USERNAME")
```

Comment:
You don't have to use `get()`. It's OK to fail hard if your environment variables are missing.